src/core/qgsmaprenderersequentialjob.cpp

```diff
@@ -31,6 +31,7 @@ QgsMapRendererSequentialJob::QgsMapRendererSequentialJob( const QgsMapSettings&
   mImage = QImage( mSettings.outputSize(), mSettings.outputImageFormat() );
   mImage.setDotsPerMeterX( 1000 * settings.outputDpi() / 25.4 );
   mImage.setDotsPerMeterY( 1000 * settings.outputDpi() / 25.4 );
+  mImage.fill( settings.backgroundColor().rgba() );
```

Comment:
Can you give some more pointers as to when this is an issue? Looking at this, it seems that the this will already be handled by QgsMapRendererCustomPainterJob::start() (line 70).